base/docker-compose.yaml

```
@@ -197,6 +197,9 @@ services:
       VITE_OIDC_LOGOUT_ENDPOINT: https://keycloak.${BASE_DOMAIN}/realms/ltt/protocol/openid-connect/logout
       VITE_OIDC_CHECK_SESSION_IFRAME: https://keycloak.${BASE_DOMAIN}/realms/ltt/protocol/openid-connect/login-status-iframe.html
       VITE_POST_LOGOUT_REDIRECT_URI: https://${BASE_DOMAIN}/users
+      VITE_LOG_URL: https://shl-server.${BASE_DOMAIN}/api/log
+      VITE_SYSTEM_URL: https://shl-creator.${BASE_DOMAIN}
+      VITE_VERSION_STRING: ${SHL_CREATOR_IMAGE_TAG:-"latest"}
```

This shouldn't need to be quoted, at least for YAML

```suggestion
      VITE_VERSION_STRING: ${SHL_CREATOR_IMAGE_TAG:-latest}
```